docs/reference/stacker_file.md

```diff
@@ -63,50 +63,42 @@ layer on a previously specified layer in the stacker file.
 
 `scratch`: which is an empty rootfs and can be used to host statically built binaries.
 
-### `import`
+### `imports`
 
-The `import` directive describes what files should be made available in
-`/stacker` during the `run` phase. There are three forms of importing supported
-today:
+The `imports` directive describes what files should be made available in `/stacker/imports` during the `run` phase. There are three forms of importing supported today:
 
     /path/to/file
 
-Will import a file or directory from the local filesystem. If the file or
-directory changes between stacker builds, it will be hashed and the new file
-will be imported on subsequent builds.
+Will import a file or directory from the local filesystem. If the file or directory changes between stacker builds, it will be hashed and the new file will be imported on subsequent builds.
 
     http://example.com/foo.tar.gz
 
-Will import foo.tar.gz and make it available in `/stacker`. Note that stacker
-will NOT update this file unless the cache is cleared, to avoid excess network
-usage. That means that updates after the first time stacker downloads the file
-will not be reflected.
+Will import foo.tar.gz and make it available in `/stacker`. Note that stacker will NOT update this file unless the cache is cleared, to avoid excess network usage. That means that updates after the first time stacker downloads the file will not be reflected.
 
     stacker://$name/path/to/file
 
 Will grab /path/to/file from the previously built layer `$name`.
```

Comment:
we should replace 'layer' with 'image' here, even though it's not a new change

Reply:
Done